pkgs/by-name/gl/glance/package.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "glance";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "glanceapp";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-37nQEpJxioELNFJxacOUWOxGMFm80UtaYLDCxsoXRe8=";
+  };
+
+  vendorHash = "sha256-Okme73vLc3Pe9+rNlmG8Bj1msKaVb5PaIBsAAeTer6s=";
+
+  meta = with lib; {
+    homepage = "https://github.com/glanceapp/glance";
+    changelog = "https://github.com/glanceapp/glance/releases/tag/v${version}";
+    description = "A self-hosted dashboard that puts all your feeds in one place";
+    mainProgram = "glance";
+    license = licenses.agpl3Only;
+    maintainers = [ ];
```

Comment:
If nobody's going to maintain it, consider adding an update script so that at least it still gets updated by a bot.
```nix
passthru.updateScript = nix-update-script { };
```

Reply:
@dvn0  Better to add yourself to the maintainer list because you've submitted other packages.

Reply:
Better to add @dvn0 to the maintainers because this isn't your first package.
https://github.com/NixOS/nixpkgs/tree/master/maintainers

Reply:
i'm okay with adding myself as the maintainer, but can i still use this update script thing?

Reply:
i've added [a commit](https://github.com/NixOS/nixpkgs/pull/310955/commits/3598249b13ddfc2b3be6e5d36c1c464a035e481b) to put me in the maintainers list, and set myself as maintainer for this package.

Reply:
> i'd be happy with becoming a maintainer, but can i still use this update script thing?

You can, and it would make your life so much easier :P A bot would automatically detect updates to packages you maintain and ping you about it, so you don't have to pay attention to upstream releases yourself that much